src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -62,6 +62,15 @@ struct mlx5_grh_av {
 #  define MLX5_WQE_CTRL_SOLICITED  (1<<1)
 #endif
 
+#define UCT_IB_MLX5_AM_ZCOPY_MAX_IOV  3UL
+
+#define UCT_IB_MLX5_AM_MAX_HDR(_av_size) \
```

Comment:
The macro `UCT_RC_MLX5_AM_MAX_HDR` moved into common place and reused in UD.
The name changed to `UCT_IB_MLX5_AM_MAX_HDR`.